h1st/model/fuzzy/fuzzy_modeler.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+from h1st.model.modeler import Modeler
+from h1st.model.fuzzy.fuzzy_model import FuzzyModel
+from h1st.model.fuzzy.fuzzy_variables import FuzzyVariables
+from h1st.model.fuzzy.fuzzy_rules import FuzzyRules
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class FuzzyModeler(Modeler):
+    """
+    FuzzyModeler is a Modeler for FuzzyModel. Using FuzzyVariables and FuzzyRules
+    FuzzyModeler builds FuzzyModel.
```

Comment:
We should explain here why this otherwise complicated run-around. Explain that it is to support the standard pattern in H1st Model where a `Modeler` is responsible for building a `Model`, which may often including loading data, training, evaluation, etc. It happens to be very simple for rule-based models like this, but we must still consistently follow that pattern.

Reply:
Noted with thanks.